app/screens/home/channel_list/categories_list/categories/categories.tsx

```diff
@@ -105,17 +112,17 @@ const Categories = ({
 
     return (
         <>
-            {!switchingTeam && !initiaLoad && onlyUnreads &&
+            {!switchingTeam && !initiaLoad && showAllCategory &&
             <View style={styles.mainList}>
                 <UnreadCategories
                     currentTeamId={teamId}
                     isTablet={isTablet}
                     onChannelSwitch={onChannelSwitch}
-                    onlyUnreads={onlyUnreads}
+                    onlyUnreads={showAllCategory}
                 />
             </View>
             }
-            {!switchingTeam && !initiaLoad && !onlyUnreads && (
+            {!switchingTeam && !initiaLoad && !showAllCategory && (
```

Comment:
Now this should read as:
- If `showOnlyUnreadsCategory`, show the unread categories.
- Else, show a list with all the different categories.

Reply:
@larkox Updated